frontend/pages/media/marketing.js

```diff
@@ -45,34 +41,30 @@ const MarketingContent = () => {
     setCurrentPage("All");
   };
 
-  // get marketing archives
-  // input: marketing archives data from google sheets
-  // output: array of dictionaries containing marketing archives data
-  const fetchMarketingArchives = async () => {
-    const res = await getMarketingArchives();
-    const tempContent = humps.camelizeKeys(res);
-    console.log(res);
-    setContent(tempContent);
-    setCurrentPosts(tempContent.slice(0, postsPerPage));
-    setSelectedPosts(tempContent);
+  // scroll to top on load
+  useEffect(() => window.scrollTo(0, 0), []);
+
+  const fetchMarketingArchive = (archives) => {
+    setContent(archives);
+    archives = archives.filter((item) => {
+      return item.fields.year === year;
+    });
+    setCurrentPosts(archives.slice(0, postsPerPage));
+    setSelectedPosts(archives);
     setLoading(false);
     setSourceLoading(false);
   };
 
-  // scroll to top on load
+  // get marketing archives
+  // input: marketing archives data from google sheets
+  // output: array of dictionaries containing marketing archives data
   useEffect(() => {
-    window.scrollTo(0, 0);
-    // load marketing previews
-    fetchMarketingArchives().catch((error) =>
-      // error handling
-      console.error(error)
-    );
-  }, []);
-
+    setLoading(true);
```

Comment:
Look into reimplementing the loading element for the timeline - it seems to be removed :(